ui/StatusQ/include/StatusQ/keychain.h

```diff
@@ -35,23 +35,24 @@ class Keychain : public QObject {
 
     bool loading() const;
 
-    Q_INVOKABLE void requestSaveCredential(const QString &reason, const QString &account, const QString &password);
-    Q_INVOKABLE void requestDeleteCredential(const QString &reason, const QString &account);
+    Q_INVOKABLE void requestSaveCredential(const QString &account, const QString &password);
+    Q_INVOKABLE void requestDeleteCredential(const QString &account);
```

Comment:
Now, with the assumption that saving and deleting do not involve popup imo we should consider making those calls synchronous as they are instantaneous. It would make usage much simpler.

Similarly for the query, async `requestHasCredential` seems not very suitable there.

Reply:
Alright, this is fair, I'll simplify 👍